requirements.txt

```diff
@@ -3,4 +3,4 @@ Django==4.1.10
 djangorestframework==3.13.1
 pytz==2022.1
 sqlparse==0.4.4
-mysql-connector-python==8.2.0
\ No newline at end of file
+mysql-connector-python==8.3.0
```

Comment:
No need to change package version.
Add EOF

Reply:
I return a package version which was requested by a project